.travis/build.sh

```diff
@@ -30,19 +30,21 @@ fi
 python setup.py build_ext --inplace $EXTRA_BUILDEXT
 BUILD_RES=$?
 
-if [ x"$KRB5_VER" = "xheimdal" ]; then
-    # heimdal can't run the tests yet, so just exit
+if [ $BUILD_RES -ne 0 ]; then
+    # if the build failed, don't run the tests
     exit $BUILD_RES
 fi
 
-if [ "$TRAVIS_OS_NAME" == "windows" ]; then
-    # Windows can't run tests yet, so just exit
-    exit $BUILD_RES
+if [ x"$KRB5_VER" = "xheimdal" ]; then
+    # heimdal can't run the tests yet, so just make sure it imports and exit
+    python -c "import gssapi"
+    exit $?
 fi
 
-if [ $BUILD_RES -ne 0 ]; then
-    # if the build failed, don't run the tests
-    exit $BUILD_RES
+if [ "$TRAVIS_OS_NAME" == "windows" ]; then
```

Comment:
Same check in both cases - can you fold it into a single conditional?

Reply:
Sure, it was two separate checks before, so I just left it as such